modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/metric/MetricCommandArg.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.commandline.metric;
+
+import org.apache.ignite.internal.commandline.argument.CommandArg;
+
+/** Represents all possible arguments for {@link MetricCommand}. */
+public enum MetricCommandArg implements CommandArg {
+
+    /** Id of the node to get metric values from. */
+    NODE_ID("--node-id");
+
+    /** Name of the argument. */
+    private final String name;
+
+    /** @param name Name of the argument. */
+    MetricCommandArg(String name) {
+        this.name = name;
+    }
+
+    /** {@inheritDoc} */
+    @Override public String argName() {
+        return name;
+    }
+
+    /** {@inheritDoc} */
+    @Override public String toString() {
+        return name;
+    }
+
+}
```

Comment:
New line is missing here, for example, and there's another empty line above that shouldn't be there